tests/identity_request.py

```diff
@@ -88,23 +88,44 @@ def _create_customer(cls, account, create_tenant=True):
 
     @classmethod
     def _create_request_context(
-        cls, customer_data, user_data, create_customer=True, create_tenant=False, is_org_admin=True
+        cls, customer_data, user_data, create_customer=True, create_tenant=False, is_org_admin=True, is_internal=False
     ):
         """Create the request context for a user."""
         customer = customer_data
         account = customer.get("account_id")
         if create_customer:
             cls.customer = cls._create_customer(account, create_tenant=create_tenant)
+
         identity = {
```

Comment:
this seems could be removed since you have replaced it with _build_identity

Reply:
Good catch, thanks! It's removed, now. 😄